docs/reference/evercrypt/aead/index.md

```diff
@@ -0,0 +1,107 @@
+# AEAD (``EverCrypt_AEAD.h``)
+
+Clients are expected to allocate persistent state first, which performs key expansion and precomputes internal data, e.g., for AES-GCM.
```

Comment:
That's a weird start to the page.
I'd recommend to start similar to the chachapoly page where you first say what this algorithm is.